src/vorta/views/diff_result.py

```diff
@@ -88,6 +37,62 @@ def parse_line(line):
         self.okButton.clicked.connect(self.accept)
 
 
+def parse_diff_lines(diff_lines):
+    files_with_attributes = []
+    nested_file_list = nested_dict()
+
+    def parse_line(line):
+        if line:
+            line_split = line.split()
+        else:
+            return 0, "", "", ""
+
+        if line_split[0] in {'added', 'removed', 'changed'}:
```

Comment:
Is it possible that you meant `['added', 'removed', 'changed']`?

Reply:
I just checked, a dict works as well, but a list or tuple still makes more sense in my opinion.

Reply:
I think using a set is as good, or maybe even a hair faster, but I'll change it to a list if that's what you prefer.

Reply:
No, it's okay.
I wasn't aware that curly braces without a colon, are equivalent to set() in Python https://docs.python.org/3/tutorial/datastructures.html#sets

Reply:
Oh, but that's not a dict - it's a [set](https://docs.python.org/3/tutorial/datastructures.html#sets).